modules/auxiliary/analyze/crack_windows.rb

```diff
@@ -37,9 +37,11 @@ def initialize
     register_options(
       [
         OptBool.new('NTLM',  [false, 'Crack NTLM hashes', true]),
-        OptBool.new('LANMAN',[false, 'Crack LANMAN hashes', true]),
-        OptBool.new('INCREMENTAL',[false, 'Run in incremental mode', true]),
-        OptBool.new('WORDLIST',[false, 'Run in wordlist mode', true])
+        OptBool.new('LANMAN', [false, 'Crack LANMAN hashes', true]),
+        OptBool.new('MSCASH', [false, 'Crack M$ CASH hashes (1 and 2)', true]),
+        OptBool.new('INCREMENTAL', [false, 'Run in incremental mode', true]),
+        OptBool.new('WORDLIST', [false, 'Run in wordlist mode', true]),
+        OptBool.new('NORMAL', [false, 'Run in normal modei (John the Ripper only)', true])
```

Comment:
"modei"?

Reply:
Oh! ugly typo, will be fixed in the next commit. So, now you know I use vim ;)